src/dataset/nuscenes_dataset.py

```diff
@@ -0,0 +1,22 @@
+from nuscenes import NuScenes
+from nuscenes.utils.splits import create_splits_scenes
```

Comment:
Imports for individual functions are discouraged by the style guide: https://google.github.io/styleguide/pyguide.html#22-imports